src/inmanta/server/services/environment_metrics_service.py

```diff
@@ -217,7 +218,7 @@ class ResourceCountMetricsCollector(MetricsCollector):
     """
 
     def get_metric_name(self) -> str:
-        return "resource_count"
+        return "resource.resource_count"
```

Comment:
@arnaudsjs this is not related to this PR but is also not right, so i'll fix it here